src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -594,6 +604,18 @@ public String getInstructorFeedbackRemindParticularStudentsLink(String courseID,
         return link;
     }
     
+    /**
+     * Retrieves the link to submit the request for remind particular student
+     * @param returnUrl the url to return to after submitting the request
+     * @return submit link with return url appended to it
+     */
+    public String getInstructorFeedbackRemindParticularStudentActionLink(String returnUrl) {
```

Comment:
If anything, we should probably follow the constants' names and rename the current `getInstructorFeedbackRemindParticularStudentsLink` to `getInstructorFeedbackRemindParticularStudentsPageLink ` instead.
"Action" does little to differentiate it since they're technically all linked to actions.
(and do make the "students" plural too)